src/Console/MigrateMakeCommand.php

```diff
@@ -59,11 +60,26 @@ public function __construct(TenantiManager $tenant, Creator $creator, Composer $
      */
     public function handle()
     {
+        $arg1 = $this->argument('driver');
+        $arg2 = $this->argument('name');
```

Comment:
I don't prefer `$arg1` and `$arg2`, might just use `$argument[0]` and `$argument[1]` (using array).